src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  let PRICE_ALL = PRICE_PER_DAY * days;
+  const SHORT_DISCOUNT = 20;
+  const LONG_DISCOUNT = 50;
+  const SHORT = 3;
+  const LONG = 7;
```

Comment:
```suggestion
  const SHORT_TERM = 3;
  const LONG_TERM = 7;
```
Short and long what?